src/components/ReactTags.js

```diff
@@ -367,13 +381,17 @@ class ReactTags extends Component {
     }
 
     // call method to add
-    this.props.handleAddition(tag);
+    if (currentEditIndex !== null && this.props.handleUpdateTag)
```

Comment:
```suggestion
    if (!currentEditIndex && handleUpdateTag)
```
`handleUpdateTag` can be destructed on line 361

Reply:
Wouldn't this cause issue when we try to edit tag at index 0?

Reply:
good point, lets make it `currentEditIndex != -1` instead